src/main/webapp/app/course/manage/course-group.component.ts

```diff
@@ -272,4 +277,35 @@ export class CourseGroupComponent implements OnInit, OnDestroy {
         this.rowClass = className;
         setTimeout(() => (this.rowClass = undefined));
     };
+    /**
+     * Method for exporting the csv with the needed data
+     */
+    exportUserInformation() {
+        if (this.allCourseGroupUsers.length > 0) {
+            const rows: any[] = this.allCourseGroupUsers.map((user: User) => {
+                const data = {};
+                data[NAME_KEY] = user.name ? user.name!.trim() : '';
+                data[USERNAME_KEY] = user.login ? user.login!.trim() : '';
+                data[EMAIL_KEY] = user.email ? user.email!.trim() : '';
+                data[REGISTRATION_NUMBER_KEY] = user.visibleRegistrationNumber ? user.visibleRegistrationNumber!.trim() : '';
+                return data;
+            });
+            const keys = [NAME_KEY, USERNAME_KEY, EMAIL_KEY, REGISTRATION_NUMBER_KEY];
+            this.exportAsCsv(rows, keys);
+        }
+    }
+    exportAsCsv(rows: any[], keys: string[]) {
```

Comment:
Missing empty line and missing documentation.

Reply:
Thanks for the comment. I have added the line and the documentation.